addons/core/functions/fnc_initCBASettings.sqf

```diff
@@ -575,18 +583,18 @@
     {tf_independent_radio_code = GVARMAIN(radiocode_independent);}
 ] call CBA_Settings_fnc_init;
 [
-    "TFAR_instantiate_instantiateAtBriefing", 
-    "CHECKBOX", 
-    [ELSTRING(settings,instantiateAtBriefing), ELSTRING(settings,instantiateAtBriefing_desc)], 
-    localize ELSTRING(settings,global), 
+    "TFAR_instantiate_instantiateAtBriefing",
+    "CHECKBOX",
+    [ELSTRING(settings,instantiateAtBriefing), ELSTRING(settings,instantiateAtBriefing_desc)],
+    localize ELSTRING(settings,global),
     false,
     1
 ] call CBA_Settings_fnc_init;
 [
-    "TFAR_defaultIntercomSlot", 
-    "SLIDER", 
-    ["Default Intercom Channel", "Default Intercom channel when entering the vehicle. Has to be number. -1 is Disabled."], 
-    localize ELSTRING(settings,global), 
```

Comment:
what are these end of line changes?

Reply:
Atom done this changes automatically

Reply:
https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/master/.editorconfig#L4
:thinking: 
Need to double check that later

Reply:
Seems like my fault. 
```
git config --global core.autocrlf
true
```